app/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
 
     namespace 'com.infomaniak.drive'
 
-    compileSdk 34
+    compileSdk 35
```

Comment:
Doesn't this cause problems because of Edge to edge ? 
We should wait for Sirambd's migration end before bumping no ?